zellij-utils/assets/config/log4rs.yml

```diff
@@ -0,0 +1,46 @@
+# Scan this file for changes every 30 seconds
+refresh_rate: 30 seconds
+
+appenders:
+  # An appender named "stderr" that writes to stderr
+  stderr:
+    kind: console
+    target: stderr
+
+  # default zellij appender, should be used across most of the codebase.
+  logFile:
+    kind: file
+    path: "zellij.log"
+    append: false
```

Comment:
Eventually, this should probably also be true?

Reply:
Having a `date` on all of the messages resolves some ambiguity and we could have some startup / shutdown messages that make it clear when Zellij is restarted.

Reply:
Sounds good!